src/control/tag.rs

```diff
@@ -30,6 +30,12 @@ impl Tag {
         self.0 & 0x01 != 0
     }
 
+    /// Checks whether a control value is EMPTY.
+    #[inline]
+    pub(crate) const fn is_empty(self) -> bool {
```

Comment:
Is there a reason why `special_is_empty` doesn't do that, then?

Reply:
Yes, there is a good reason for having it implemented this way. In the insertion code we use this code:
```rust
self.table.growth_left -= old_ctrl.special_is_empty() as usize;
```
Because `special_is_empty` only looks at the low bit, this becomes branchless and compiles down to 2 instructions: `and` & `sub`.